app/frontend/helpers/formatAccountIndex.ts

```diff
@@ -0,0 +1 @@
+export const formatAccountIndex = (accountIndex: number) => `#${accountIndex + 1}`
```

Comment:
it seems like an overkill to me to create a file just to declare this helper - I'd consider adding it to an index.ts file together with other such trivial helpers

Reply:
Do you min addressing this in another PR? This would also go for other helpers which is a bit out of scope for this PR.